docs/pages/server-access/architecture.mdx

```diff
@@ -0,0 +1,228 @@
+
+Our [Getting Started](./getting-started.mdx) guide demonstrates an example where a Teleport admin has temporary root access to a Linux cloud instance (through SSH port `22`) which is then removed. Further access is only possible through the Teleport Web Interface or `tsh`.
+
+For example, consider the user `tele-admin` below:
+
+```
+# Add a user
+sudo tctl users add tele-admin --roles=editor,access --logins=root,ubuntu,ec2-user
+```
+
+Note that the `--role` flag above does not give `tele-admin` the actual `admin` role. 
+
+Note further that the `--logins` flag defines what `tele-admin` can do or perform once they've connected to a resource through `tsh`.
+
+### SSH Bastion Pattern
+
+Jump Hosts and Bastion Hosts are common in use since they allow access to resources without requiring those resources to be publicly available.
+
+A common approach involves restricting the open ports to only the most minimal ones required. Our [Getting Started](./getting-started.mdx) guide demonstrates how to close *all* ports for a protected resource while still granting access through Teleport.
+
+<Figure
+  align="center"
+  bordered
+  caption="Teleport Bastion Pattern"
+>
+  ![Teleport Bastion Pattern](../../img/server-access/bastion-pattern.svg)
+</Figure>
+
+Teleport Nodes can easily be configured into the SSH Bastion Host pattern as described in the [Getting Started](./getting-started.mdx) guide.
+
+Generally, one begins by creating or provisioning resources with the minimal necessary open ports. One such resource will serve as the designated public-facing *Bastion Host* through which all traffic into the cluster will be received. The other resources will only be accessible through the *Bastion Host*.
+
+Once Teleport has been installed, Nodes connected, and initial users and permission created any remaining ports can be closed on both the *Bastion Host* and the protected, private, resources.
+
+### Divided topology
+
+Many architectures are now being designed to mitigate security events or disasters by splitting up resources and permissions into minimal groupings.
+
+Divided topologies involve deliberately restricting traffic, filtering IP addresses, regulating permissions, and grouping resources to mitigate loss or access in the advent of a disaster or security event.
+
+Teleport divides and groups access across resources through [Labels](../admin-guide.mdx#labeling-nodes-and-applications) and Node-based clustering. 
+
+- Admins can dynamically modify which Nodes join or are a part of which cluster.
+- Labels provide convenient groupings when querying, introspecting, or visualizing resources without modifying underlying clusters or access.
+
+## Cluster introspection
+
+Teleport simplifies managing and introspecting your cluster.
+
+### Unified Resource Catalog
```

Comment:
```suggestion
### Resource Catalog
```

I would drop unified here, to be inline with dropping it from Unified Access Plane